fling/component/client/cross_domain_base_client.py

```diff
@@ -0,0 +1,233 @@
+import copy
+import random
+
+import torch
+from torch.utils.data import DataLoader, Dataset
+import torch.nn as nn
+
+from fling.utils import get_optimizer, VariableMonitor, get_weights
+from fling.utils.registry_utils import CLIENT_REGISTRY
+from .client_template import CrossDomainClientTemplate
+
+
+@CLIENT_REGISTRY.register('cross_domain_base_client')
+class CrossDomainBaseClient(CrossDomainClientTemplate):
```

Comment:
Almost the same as ``BaseClient`` ? Inherit the ``BaseClient`` and only do some necessary modifications.